.template-lintrc.js

```diff
@@ -6,9 +6,7 @@ module.exports = {
 
   rules: {
     'no-bare-strings': true,
-    'quotes': 'double'
+    quotes: 'double',
```

Comment:
Usually we prefer single quotes right? Is Ember enforcing double quotes? any reason why?

Reply:
In templates, the recommended convention for quotes is double ji. Only in js/ts we default to single quotes.